src/content/docs/guides/platform-management/configure-workspaces.mdx

```diff
@@ -14,13 +14,80 @@ You can configure workspaces in two ways:
 
 ## Manage workspaces dynamically
 
-Our [platform command-line utility](/reference/platform/command-line-interface)
-supports managing at runtime.
+The Tenzir Platform CLI allows administrators to create, modify, and delete
+workspaces on demand.
 
-:::caution[WIP]
-Extract content from CLI docs to here.
+:::caution[On-premise setup required]
+This CLI functionality requires an on-premise platform deployment, available
+with the [Sovereign Edition](https://tenzir.com/pricing).
 :::
 
+Only local platform administrators can manage workspaces dynamically. The
+[`TENZIR_PLATFORM_OIDC_ADMIN_RULES` variable](/guides/platform-setup/configure-identity-provider)
+defines who's an administrator in your platform deployment.
+
+### Creating and managing workspaces
+
+You can create workspaces for either individual users or organizations. When
+you create a user workspace, it's automatically configured with access for that
+specific user. Organization workspaces start with no access rules, giving you
+full control over who can access them.
+
+For a detailed overview, of the available commands, take a look at our
+[CLI reference documentation](/reference/platform/command-line-interface).
+
+### Example: Setting up an organization workspace
+
+Let's walk through creating a workspace for the fictional "Scrooge & Marley Counting House" organization:
+
+1. **Create the organization workspace:**
```

Comment:
Can you use Starlight's `Steps` component here? That would make it look nicer.

Reply:
Done! Replaced the numbered list with Starlight's Steps component.